src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -512,6 +518,37 @@ private static bool SourceIsAttachedProperty(INamedTypeSymbol type, string name)
 			} while (true);
 		}
 
+		/// <summary>
+		/// Get the type of the attached property.
+		/// </summary>
+		private INamedTypeSymbol GetAttachedPropertyType(XamlMemberDefinition member)
+		{
+			var type = GetType(member.Member.DeclaringType);
+			return _getAttachedPropertyType(type, member.Member.Name);
+		}
+
+		private static INamedTypeSymbol SourceGetAttachedPropertyType(INamedTypeSymbol type, string name)
+		{
+			do
+			{
+				var property = type.GetAllProperties().FirstOrDefault(p => p.Name == name);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'property'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)